deployment/ansible/roles/model-my-watershed.geoprocessing/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Update CA Certificates
+  shell: "update-ca-certificates -f"
```

Comment:
This issue is unfamiliar to me; we use OpenJDK with the AWS SDK on a few other projects and I don't think I've seen it before. Before I give the OK on this solution I'd like to take a deeper look at the error locally. I think I remember how to reproduce this from our conversation on Friday.